src/app/domain/geolocation/tracking/index.js

```diff
@@ -80,20 +95,41 @@ export const startTracking = async () => {
         smallIcon: 'mipmap/ic_stat_ic_notification'
       }
     })
-    if (!state.enabled) {
-      await BackgroundGeolocation.start()
-      Log('Tracking started')
-    }
+
+    Log('State after init: ' + JSON.stringify(state))
+
     await storeData(
       CozyPersistedStorageKeys.ShouldBeTrackingFlagStorageAdress,
       true
     )
```

Comment:
This should not be stored here but in the real `startTracking` function.

Reply:
Absolutely, I realized that in parallel. Good catch!